source/config.js

```diff
@@ -235,6 +239,25 @@ if (typeof currentRootPath !== 'string') {
 }
 module.exports.rootPath = currentRootPath;
 
+let currentGitCommand = module.exports.gitCommand;
+if (typeof currentGitCommand !== 'string') {
+  currentGitCommand = '';
+} else if (currentGitCommand !== '') {
+  // must start with a slash
+  if (currentGitCommand.charAt(0) !== '/') {
+    currentGitCommand = '/' + currentGitCommand;
+  }
+  // can not end with a trailing slash - it should be a command
+  if (currentGitCommand.charAt(currentGitCommand.length - 1) === '/') {
+    currentGitCommand = currentGitCommand.substring(0, currentGitCommand.length - 1);
+  }
+}
+// finally set default if empty
+if (currentGitCommand === '') {
```

Comment:
As for paths - on windows node accepts either `D:\Some\Path` or `D:/Some/Path` (double backslashes are to escape special char, which backslash is), also AFAIK on windows node will strip leading slash.

> Your example will result in `/D:\\Apps\\Git\\bin\\git` ...

Actually it will be `/D:\Apps\Git\bin\git` inside the code - if not escaped in config string will become `D:AppsGiingit`